ui/app/components/transaction-list-item.js

```diff
@@ -44,27 +44,43 @@ TransactionListItem.prototype.render = function () {
 
   return (
     h(`.transaction-list-item.flex-row.flex-space-between${isClickable ? '.pointer' : ''}`, {
-      onClick: (event) => {
-        if (isPending) {
-          this.props.showTx(transaction.id)
-        }
-
-        if (!transaction.hash || !isLinkable) return
-        var url = explorerLink(transaction.hash, parseInt(network))
-        extension.tabs.create({ url })
-      },
       style: {
         padding: '20px 0',
       },
     }, [
 
       // large identicon
       h('.identicon-wrapper.flex-column.flex-center.select-none', [
-        transaction.status === 'unconfirmed' ? h('i.fa.fa-ellipsis-h', {style: { fontSize: '27px' }})
-         : h(TransactionIcon, { txParams, transaction, isTx, isMsg }),
+        transaction.status === 'unconfirmed' ? h('i.fa.fa-ellipsis-h', {
+          style: {
+            fontSize: '27px',
+          },
+        }) : h( '.pop-hover', {
+          onClick: () => {
```

Comment:
Here is where you could receive the `event` argument to call `stopPropagation()` on.